recipes/actflow/LICENSE

```diff
@@ -0,0 +1,49 @@
+The Brain Activity Flow Toolbox (Actflow Toolbox),
```

Comment:
@SammedAdmuthe I'm not a laywerlaywer and it is complicated for me to merge custom licenses. If you can ask upstream to publish this under an OSI approved license it would make merging much easier.

For example, the second last paragraph:

```
No part of the Software may be reproduced, modified, transmitted or
transferred in any form or by any means, electronic or mechanical,
without the express permission of the University.
```

Can be interpreted as redistribution, meaning conda-forge cannot publish this without permission.
@conda-forge/core what do you think?

Reply:
Agreed. We cannot ship this as is.

Reply:
Changed License to OSI approved MIT license